# Executes joints buffer
---
bool success
string msg
